### ISSUE TITLE
Combine Stops Harvesting

### ISSUE THREAD

**Issue by lcc1211** [Courseplay/courseplay#4629]
I have seen this issue pop up every now and then, but not too often. It happened again today with the latest CP release. The harvester filled to about 5% and kept going around the headland but didn't pick up any more of the fruit. I checked to make sure it was all the same (oats), and it was. I then checked to make sure the growth stage was 100% and it was. Please let me know if you debug channels on. 
This was on Lone Oak Farm field 41 using in game John Deere.
[log.txt](https://github.com/Courseplay/courseplay/files/3929198/log.txt)

**Comment by Tensuko** [comment]
Remove all other Maps from your mods Folder.
This is an Important Rule: Only one Mod Map in the Mods Folder at the same time!
Try it and report, that might be the Issue here.

**Comment by lcc1211** [comment]
I took all the maps out except for the Lone Oak and tried to duplicate on the several fields I already owned. I can't repeat the issue as this happens on a very rarely. However, there is something that tends to accompany it that I just remembered. One vehicle of various kinds will no longer show the CP interface no matter how I try to view it. It will work on all the others just fine. Always a mod vehicle though. I'll close this one and try to capture a log when they both happen together again.
